src/app/libp2p_helper/Makefile

```diff
@@ -14,7 +14,7 @@ libp2p_helper: ../../libp2p_ipc/libp2p_ipc.capnp.go
 test: ../../libp2p_ipc/libp2p_ipc.capnp.go
 	cd src/libp2p_helper \
 		&& (ulimit -n 65536 || true) \
-		&& $(GO) test -short -timeout 40m
+		&& $(GO) test -short -timeout 60m
```

Comment:
wow, that _is_ short 😁 

is this timeout justified by having to run more tests?
could it have been accidentally leaked from development process?

Reply:
The 40 minute timeout wasn't enough for these changes with some runs. I found relatively low variance with a 60 second timeout. I wasn't sure what the problem was initially on remote until i got the test passing locally after waiting longer. We run The new downloader test as well as the util test. It seams justified to me, because the downloader was not required before.